web-client/src/views/AdvancedSearch/OpinionSearchForm.tsx

```diff
@@ -38,13 +38,7 @@ export const OpinionSearchForm = connect(
   }) {
     return (
       <>
-        <form
-          data-testid="opinion-search-container"
-          onSubmit={e => {
-            e.preventDefault();
-            submitAdvancedSearchSequence();
-          }}
-        >
```

Comment:
Just for others following along: if we set the validation on the form like this and call `onSubmit` in the buttons, USWDS errors will prevent the form submission, which also prevents us from running our own validation sequence (and we don't want that!).